Documentation/migration-scripts/python-scripts/remapping_users_to_groups.py

```
@@ -0,0 +1,237 @@
+            # Skip system/admin protected groups (optional safety)
+            if m["group_id"] == 2:  # Administrators
+                continue
+
+            remove_membership(url, session_id, m["membership_id"])
+
+        # Add new groups
+        for gid in updated_ids:
+            if gid != 1:
+                try:
+                    add_user_to_group(url, session_id, new_user["id"], gid)
+                except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)